neuralpredictors/layers/readouts/gaussian.py

```diff
@@ -91,9 +94,8 @@ def initialize(self):
                 "fixed_sigma to True"
             )
         self.features.data.fill_(1 / self.in_shape[0])
-
         if self.bias is not None:
-            self.bias.data.fill_(0)
+            self.initialize_bias(mean_activity=mean_activity)
```

Comment:
What happens if `mean_activity` is passed but `bias=None`?

Reply:
Every individual readout has this line in the initialize method, so it's taken care of:
if self.bias is not None:
    self.initialize_bias(mean_activity=mean_activity)

Reply:
But if someone passes the `mean_activity` without setting `bias=True` then it is gonna be ignored, right? I mean, in this scenario even though the user has passed the `mean_activity` the bias would be initialized randomly (or whatever the default is for Pytorch)
Also, now that I am thinking about this, what are the expected values of `bias`? is it supposed to be a boolean?

Reply:
If bias=False, there will be no initialization with 0 because there will be no bias. The argument `bias` to the `__init__` is boolean and says whether there is a bias or not. `self.bias` in the initialize, however, is either `None` (if `bias` was `False`) or a `torch.tensor` (if `bias` was `True`). `if self.bias is not None` (i.e. if `bias` was `True`), the so existing bias will be initialized with the mean activity (which defaults to None and then results in initialization with 0)